src/test/kotlin/blackjack/domain/model/DealerTest.kt

```diff
@@ -0,0 +1,47 @@
+package blackjack.domain.model
+
+import blackjack.domain.PointCalculator
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+class DealerTest {
+    @Test
+    fun `딜러의 이름은 딜러이다`() {
+        val dealer = Dealer(Game())
+        dealer.info.name shouldBe "딜러"
+    }
+
+    @Test
+    fun `딜러는 처음에 카드 2장을 가지고 있다`() {
+        val dealer = Dealer(Game())
+        dealer.cards.items.size shouldBe 2
+    }
+
+    @Test
+    fun `딜러가 가지고 있는 카드의 합을 구한다`() {
+        val dealer = Dealer(Game())
+        val cards = dealer.cards
+        val expectedSum = PointCalculator.sum(cards)
+
+        dealer.cards.sum shouldBe expectedSum
+    }
+
+    @Test
+    fun `딜러가 가지고 있는 카드의 합이 17이 되지 않으면 카드를 계속 뽑을 수 있다`() {
```

Comment:
```suggestion
    fun `딜러가 가지고 있는 카드의 합이 17 미만이면 카드를 한 장 뽑는다`() {
```
17이 될 때까지 계속 뽑을 수 없고, 한 장만 뽑습니다. 😅